addons/point_of_sale/static/src/js/Screens/TicketScreen/TicketScreen.js

```diff
@@ -148,6 +148,9 @@ export class TicketScreen extends IndependentToOrderScreen {
         }
         const { globalState } = this.pos;
         if (order && (await this._onBeforeDeleteOrder(order))) {
+            if (Object.keys(order.lastOrderPrepaChange)) {
```

Comment:
Do you intend to compare the length to zero like so: `Object.keys(order.lastOrderPrepaChange).length !== 0)`?